ls1intum/Artemis
# `Development`: Fix grading instruction values sent as strings to the server

### Checklist
#### General
#### Server
#### Client

### Motivation and Context
Follow-up for #6098.

This fixes the cause for having to parse strings to doubles in the server. The usage of `any` allowed to assign strings to a number-attribute.

### Description
The method `updateGradingInstructionProperty` was unnecessary. Specifying the ` [(ngModel)]` is enough and will automatically update the underlying object. 

This PR also fixes a small visual issue on the re-evaluate confirmation buttons (see Screenshots below)

### Steps for Testing
Prerequisites:
- 1 Instructor

1. Create a new Exercise with Structured Grading Instructions -> Saving works
2. Update credits and usage count of the instruction and save the exercise -> Re-evaluate window appears and saving works.

### Review Progress
#### Code Review
#### Manual Tests
#### Exam Mode Test

### Test Coverage
unchanged

### Screenshots
old window:
![grafik](https://user-images.githubusercontent.com/26540346/213457522-ce5f3256-b07f-469a-bd87-478974a272f1.png)

new window: 
![grafik](https://user-images.githubusercontent.com/26540346/213457651-badc33a9-fa2d-4a22-ad5b-9feb6f8a46f1.png)
